js/src/site.js

```diff
@@ -14,6 +14,16 @@
 			scrollTop = 0,
 			windowHeight = 0,
 			windowWidth = 0,
+			min_xs = "@mq_min_xs",
```

Comment:
Vars that aren't local should be camel case -- I'd also advocate making them more readable rather than shorter (i.e. minTiny, minSmall, minMedium, minLarge, minHuge or something along those lines). Maybe Smallest/Largest rather than trying to give them scale names, haha.